src/Tools/AI Test Toolkit/src/Logs/AITLogEntry.Table.al

```diff
@@ -0,0 +1,283 @@
+        field(18; "Run ID"; Guid)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Run ID';
+        }
+        field(20; "Original Operation"; Text[100])
+        {
+            Caption = 'Original Operation';
+        }
+        /// <summary>
+        /// Contains the original status of the test if any event subscribers modifies the status of the test
+        /// </summary>
+        field(21; "Original Status"; Option)
+        {
+            Caption = 'Original Status';
+            OptionMembers = Success,Error;
+        }
+        /// <summary>
+        /// Contains the original message of the test if any event subscribers modifies the message of the test
+        /// </summary>
+        field(22; "Original Message"; Text[250])
+        {
+            DataClassification = CustomerContent;
+            Caption = 'Original Message';
+        }
+        /// <summary>
+        /// Is true if any event subscribers has modified the log entry
+        /// </summary>
+        field(23; "Log was Modified"; Boolean)
+        {
+            Caption = 'Log was Modified';
+        }
+        field(24; "Test Input Group Code"; Code[100])
+        {
+            Caption = 'Test Input Group Code';
+            TableRelation = "Test Input Group".Code;
+        }
+        field(25; "Test Input Code"; Code[100])
+        {
+            Caption = 'Test Input Code';
+            TableRelation = "Test Input".Code where("Test Input Group Code" = field("Test Input Group Code"));
+        }
+        field(26; "Test Input Description"; Text[2048])
+        {
+            Caption = 'Test Input Description';
+            TableRelation = "Test Input Group"."Description" where("Code" = field("Test Input Group Code"));
+        }
+        field(27; Sensitive; Boolean)
+        {
+            Caption = 'Sensitive';
+        }
+        field(28; "Input Data"; Blob)
+        {
+            Caption = 'Input Data';
+        }
+        field(29; "Output Data"; Blob)
+        {
+            Caption = 'Output Data';
+        }
+        field(40; ModelVersion; Option)
+        {
+            Caption = 'AOAI Model Version';
+            OptionMembers = Latest,Preview;
+            DataClassification = SystemMetadata;
+        }
+    }
+
+    keys
+    {
+        key(Key1; "Entry No.")
+        {
+            Clustered = true;
+        }
+        key(Key2; "Test Suite Code", Version, "Test Method Line No.", Operation, "Duration (ms)", "Test Input Code")
+        {
+            // Instead of a SIFT index. This will make both inserts and calculations faster - and non-blocking
+            IncludedFields = "Procedure Name", Status;
+        }
+        key(Key4; "Test Suite Code", Version, Operation, "Duration (ms)")
+        {
+            // Instead of a SIFT index. This will make both inserts and calculations faster - and non-blocking
+        }
+        key(Key3; "Duration (ms)")
+        {
+            SumIndexFields = "Duration (ms)";
```

Comment:
According to the comment, you do not want a SIFT index here, yet you have `SumIndexFields` specified which will create a SIFT index